src/Files.App/Data/EventArguments/ItemLoadStatusChangedEventArgs.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Files.App.Data.EventArguments
+{
+	public class ItemLoadStatusChangedEventArgs : EventArgs
+	{
+		public enum ItemLoadStatus
+		{
+			Starting,
+
+			InProgress,
+
+			Complete
```

Comment:
```suggestion
			Starting,
			InProgress,
			Complete
```

Reply:
I will leave because I will add xml comments to there.